zeppelin-plugins/notebookrepo/azure/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java

```diff
@@ -33,7 +33,7 @@
 import java.util.List;
 import java.util.Map;
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.StringUtils;
```

Comment:
Could we remove it from maven dependencies so that we force user to use lang3 ?

Reply:
I don't think that's possible at the moment. Hadoop 2.7 and the [commons-configuration](https://github.com/apache/commons-configuration/blob/CONFIGURATION_1_10/pom.xml#L211) use the old library.
At the moment I am working on migrating Zeppelin to [commons-configuration2](https://github.com/Reamer/zeppelin/tree/configuration_version_2) because the current library is quite old.